usmqe_tests/api/gluster/test_volume.py

```diff
@@ -65,8 +65,7 @@ def test_create_volume_invalid(
     # TODO check correctly server response or etcd job status
     api.wait_for_job_status(
             job_id,
-            status="failed",
-            issue="https://github.com/Tendrl/tendrl-api/issues/33")
+            status="failed")
```

Comment:
I agree, a separate pull request makes sense for stuff like this, as long as it could be merged fast - which I guess here it's the case. But this is just a note for future.